tensorflow/compiler/mlir/tensorflow/tests/shape_inference.mlir

```diff
@@ -1045,4 +1045,94 @@ module attributes {tf.versions = {bad_consumers = [], min_consumer = 0 : i32, pr
     // CHECK-SAME: tensor<i32>
     return %arg0 : tensor<*xi32>
   }
+
+  // Test conv2d inferReturnTypes.
```

Comment:
Nit: Could you modify this sentence to make a complete sentence?

Reply:
See if this is better 😃